packages/react/src/components/Loader/Loader.tsx

```diff
@@ -15,19 +15,14 @@ import { Accessibility } from '../../lib/accessibility/types'
 import { WithAsProp, ShorthandValue, withSafeTypeForAs } from '../../types'
 import Box, { BoxProps } from '../Box/Box'
 
-export type LoaderPosition = 'above' | 'below' | 'start' | 'end'
```

Comment:
Isn't @layershifter just inlining this?

Reply:
Technically it's breaking change, will mention this in `CHANGELOG` and in a PR header.
It's necessary to avoid manual type definition for autogenerated playground.

Reply:
oh, we were exporting this type 🤦‍♂  great catch @miroslavstastny